custom_components/wyzeapi/strings.json

```diff
@@ -27,6 +27,11 @@
   },
   "options": {
     "step": {
+      "init":{
+        "data":{
+          "bulb_local_control": "Use Local Control for Color Bulbs and Light Strips"
```

Comment:
This should reference the translation file like the others do.

Reply:
Look at line 36 for an example of what this should look like. You have to refer to the key in this file and then it will pull the translation from the translations file.

Reply:
I see what you're saying, but as far I can tell the keys that are referenced in all the other fields refer to the HA common strings that are stored here: https://github.com/home-assistant/core/blob/dev/homeassistant/strings.json. 

In testing I can completely delete the `stings.json` file in this integration and nothing changes until I change a translation file, which makes me think that this file is only a guide for the translations file(s). Not trying to argue here you know way more about this than I do, I just don't know what to put in the fields.

As another example of this see the Rachio component: https://github.com/home-assistant/core/blob/dev/homeassistant/components/rachio/strings.json